### PR TITLE
Implement tokens for CSO and SLO DashDotDB types

### PR DESCRIPTION
**Pull request by rrati**
This is dependent upon 
https://github.com/app-sre/dashdotdb/pull/46

### CODE REVIEW

File: reconcile/dashdotdb_base.py [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+        self.dry_run = dry_run
+        self.thread_pool_size = thread_pool_size
+        self.settings = queries.get_app_interface_settings()
+        self.secret_reader = SecretReader(settings=self.settings)
+        self.secret_content = self.secret_reader.read_all(
+            {'path': DASHDOTDB_SECRET})
+        self.dashdotdb_url = self.secret_content['url']
+        self.dashdotdb_user = self.secret_content['username']
+        self.dashdotdb_pass = self.secret_content['password']
+        self.logmarker = marker
```
Comment (BumbleFeng) [inline reconcile/dashdotdb_base.py:28]: I think we could init and verify the scope and set to self.scope here

**Review by BumbleFeng — APPROVED** [review]

File: reconcile/dashdotdb_base.py (hunk 1 above)
Comment (rrati) [reply reconcile/dashdotdb_base.py:29]: Yes, good idea.  Done

File: reconcile/dashdotdb_base.py [hunk 2]
```diff
@@ -15,62 +15,64 @@
...
         self.dry_run = dry_run
         self.thread_pool_size = thread_pool_size
         self.settings = queries.get_app_interface_settings()
         self.secret_reader = SecretReader(settings=self.settings)
         self.secret_content = self.secret_reader.read_all(
             {'path': DASHDOTDB_SECRET})
-        self.dashdotdb_url = self.secret_content['url']
-        self.dashdotdb_user = self.secret_content['username']
-        self.dashdotdb_pass = self.secret_content['password']
+        self.dashdotdb_url = 'http://localhost:8080'
```
Comment (BumbleFeng) [inline reconcile/dashdotdb_base.py:25]: I think you forget to revert the change of using localhost dddb
